build/scripts/cleanup_perf.ps1

```diff
@@ -13,25 +16,49 @@ try {
 }
 catch {}
 try {
+    taskkill /F /IM csc.exe 2>&1 | Out-Null
+}
+catch {}
+try {
+    taskkill /F /IM vbc.exe 2>&1 | Out-Null
+}
+catch {}
+try {
```

Comment:
We should make this a function to avoid duplicating the try / catch over and over again.